rpa-worker-application/src/main/resources/META-INF/application.sample.properties

```diff
@@ -0,0 +1,41 @@
+## Camunda RPA Worker
+## Full properties reference: https://github.com/camunda/rpa-worker?tab=readme-ov-file#configuration-reference
+
+### General Configuration
+#camunda.rpa.zeebe.worker-tags=default
+#camunda.rpa.robot.default-timeout=PT5M
+#camunda.rpa.python.extra-requirements=/path/to/extra-requirements.txt
+
+### Zeebe Configuration
+
+#### SaaS Production
+#camunda.client.zeebe.enabled=true
+#camunda.client.mode=saas                           #} 
+#camunda.client.cluster-id=<cluster id>             #} Replace with autogenerated properties
+#camunda.client.region=<region>                     #} from Console
+#camunda.client.auth.client-id=<client id>          #} 
+#camunda.client.auth.client-secret=<client secret>  #} 
+##### Secrets Configuration
+#camunda.rpa.secrets.backend=camunda
+
+#### Saas Dev
+#camunda.client.zeebe.enabled=true
+#camunda.client.mode=selfmanaged
+#camunda.client.cluster-id=<cluster id>             #}
+#camunda.client.region=<region>                     #} Replace with autogenerated properties
+#camunda.client.auth.client-secret=<client secret>  #} from Console (exclude camunda.client.mode which
+#camunda.client.auth.client-id=<client id>          #} should be selfmanaged for Dev)
+#camunda.client.auth.issuer=https://login.cloud.dev.ultrawombat.com/oauth/token
+#camunda.client.zeebe.rest-address=https://${camunda.client.cluster-id}.${camunda.client.region}.zeebe.dev.ultrawombat.com
+#camunda.client.zeebe.grpc-address=grpcs://${camunda.client.cluster-id}.${camunda.client.region}.zeebe.dev.ultrawombat.com
+#camunda.client.zeebe.base-url=https://${camunda.client.region}.zeebe.dev.ultrawombat.com/${camunda.client.cluster-id}
+#camunda.client.zeebe.audience=zeebe.dev.ultrawombat.com
+#camunda.rpa.zeebe.auth-endpoint=https://login.cloud.dev.ultrawombat.com/oauth
```

Comment:
This makes sense for our internal development, but it might be confusing for the customers. `ultrawombat` is Camundas internal staging environment, so external users would never encounter this URL.

How about a SaaS and a Self managed example instead?